apache-Dockerfile-block-2

```diff
@@ -1,5 +1,50 @@
 COPY apache2-foreground /usr/local/bin/
 WORKDIR /var/www/html
 
+# Set up FPM
+RUN set -ex \
+	&& cd /usr/local/etc \
+	&& if [ -d php-fpm.d ]; then \
+		# for some reason, upstream's php-fpm.conf.default has "include=NONE/etc/php-fpm.d/*.conf"
+		sed 's!=NONE/!=!g' php-fpm.conf.default | tee php-fpm.conf > /dev/null; \
+		cp php-fpm.d/www.conf.default php-fpm.d/www.conf; \
+	else \
+		# PHP 5.x doesn't use "include=" by default, so we'll create our own simple config that mimics PHP 7+ for consistency
```

Comment:
php 5.x not supported, remove the `else` block?

Reply:
(See other comment)